_gsocproposals/2022/proposal_CERNBox-Etherpad.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Etherpad plugin as a ScienceMeshDocs editor
+layout: gsoc_proposal
+project: CERNBox
+year: 2022
+organization:
+  - CERN
+  - WWU
+  - ScienceMesh
+---
+
+## Description
+
+[CERNBox](https://cernbox.web.cern.ch) is a sync and share, collaborative cloud storage solution built at CERN on top of EOS. The service is used by more than 37K users and stores over 15PB of data. CERNBox has responded to the high demand in our diverse community to an easily and accessible cloud storage solution, which provides integrations with other CERN services for Big Science: visualisation tools, interactive data analysis and real-time collaborative editing. For the latter, a number of applications have already been integrated, by leveraging the [WOPI](https://docs.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest) open specifications.
+
+[ScienceMesh](https://cs3mesc4eosc.eu) is an emerging pan-European federated cloud infrastructure, which aims at bringing together several sync and share platforms across Europe. In the context of ScienceMesh the integration of CodiMD has been demonstrated through WOPI _bridge_ extensions.
```

Comment:
I believe this is not the link you want to have...

Reply:
Thanks Benedikt and good point, we now have https://sciencemesh.io that is more appropriate than the institutional EU project. Will also fix the other comments.